k256/src/arithmetic.rs

```diff
@@ -53,6 +56,12 @@ impl PartialEq for AffinePoint {
     }
 }
 
+impl Default for AffinePoint {
+    fn default() -> AffinePoint {
+        AffinePoint::generator()
+    }
+}
```

Comment:
As [mentioned above](https://github.com/RustCrypto/elliptic-curves/pull/49/files#r442975417), this was necessitated by the current usage of `CtOption`. I'd be interested in finding a workaround so it isn't necessary.

That said, this implementation seems reasonable to me...

Reply:
It doesn't matter specifically for `CtOption`'s usage, but in all my implementations, `Default` is the identity, not the generator. This seems like it could quickly cause issues...

Reply:
Sounds good, I can change it to the identity

Reply:
Err, seems `identity()` is only defined for `ProjectivePoint`

Reply:
I eliminated the `Default` bound and deleted the `impl Default for AffinePoint` blocks